modules/switch/react/lib/Switch.tsx

```diff
@@ -21,14 +21,14 @@ const translateLength = spacing.s;
 
 const SwitchContainer = styled('div')({
   position: 'relative',
-  height: switchTapArea,
+  height: spacing.m,
   width: switchTapArea,
 });
 
 const SwitchInput = styled('input')<SwitchProps>(
   {
     position: 'absolute',
-    height: switchTapArea,
+    height: spacing.m,
```

Comment:
This decreases the tappable area by 8px. Why was this needed?

Reply:
This is to make sure `Switch` has an height of 24px total and not 32px, so a form field row has the same height than a checkbox when the label is on the left. This is what it looks like with the original height:

<img width="764" alt="image" src="https://user-images.githubusercontent.com/723109/66222543-66c6bd80-e69f-11e9-90e1-71e05067502d.png">

Reply:
Right, sorry - I was thinking checkbox was at 32 as well